### PR TITLE
Adjust brightness of SVG lights

### PR DESCRIPTION
**Pull request by anlexmatos**
I use Kitchen Sink often, and found the new v2 lit buttons to be uncomfortably bright when Rack brightness is reduced-- especially the white ones.

This PR scales the brightness (alpha) of these SVG-lights from maximum to a hard-coded minimum as Rack brightness is reduced from maximum to zero. This way, the SVGs render as normal when the "room lights" are bright, and as the room lights dim the "underlying LED brightness" of the buttons is revealed.

This roughly approximates the behavior of a frosted surface with soft back-lighting; in daylight conditions, the ambient light reflected by the surface is greater than the light produced from the LED.

### CODE REVIEW

**Comment by anlexmatos** [comment]
Implementing this revealed some odd things that maybe could be improved, for example:

- The run light-button on Seq++ is a different class from the other two buttons adjacent
- The pill-shaped lights on Kitchen Sink are a different class than those on Comp II, but they are the same as one of the box-shaped button classes on Seq++
- The buttons on Mixer8 are a different class from all the rest

Basically, the class structure for these particular button components could probably be simplified or homogenized to make future maintenance easier.

**Comment by kockie69** [comment]
I have merged it, I like the effect. I think I will apply it to the other modules to.

 As soon as v2 is stable I am planning to have a thorough look at the complete codeset to optimise/improve but for now this has no priority.

Thanks
